src/itinerary/Itinerary.tsx

```diff
@@ -100,6 +109,7 @@ const Itinerary = ({
               itemProp="location"
               itemScope
               itemType="http://schema.org/Place"
+              aria-label={place.stepLabel}
```

Comment:
What happens if `place.stepLabel` is undefined? 
Also, `stepLabel` should probably be in our Place definition (https://github.com/blablacar/ui-library/blob/master/src/typings/place.d.ts)